### ISSUE TITLE
StreamIT test failing

### ISSUE THREAD

**Issue by zalokhan** [CorfuDB/CorfuDB#520]
Stream integration test failing on assert.
```
Running org.corfudb.integration.StreamIT
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.079 sec <<< FAILURE! - in org.corfudb.integration.StreamIT
simpleStreamTest(org.corfudb.integration.StreamIT)  Time elapsed: 1.009 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[fals]e> but was:<[tru]e>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at org.corfudb.integration.StreamIT.simpleStreamTest(StreamIT.java:48)
```

**Comment by Maithem** [comment]
@zalokhan  I guess the contract of this api has changed. The api contract reads : 
"   /** Returns whether or not there are potentially more entries in this
     * stream - this function may return true even if there are no entries
     * remaining, as addresses may have been acquired by other clients
     * but not written yet, or the addresses were hole-filled, or just failed.
     * @return      True, if there are potentially more entries in the stream."

Not sure how useful this is without having a guarantee

**Comment by slfritchie** [comment]
Mid-June issues review: fixed by #586
